system/jlib/jsocket.cpp

```diff
@@ -136,7 +136,7 @@ static bool IP6preferred=false;         // e.g. for DNS and socket create
 IpSubNet PreferredSubnet(NULL,NULL);    // set this if you prefer a particular subnet for debugging etc
                                         // e.g. PreferredSubnet("192.168.16.0", "255.255.255.0")
 
-static atomic_t pre_conn_unreach_cnt = ATOMIC_INIT(0);    // global count of pre_connect() JSE_NETUNREACH error
+static std::atomic_uint pre_conn_unreach_cnt{0};    // global count of pre_connect() JSE_NETUNREACH error
```

Comment:
Could possibly be.  Erring on the side of safety.  @mckellyln is this likely to be worth changing?